project/scripts/release

```diff
@@ -162,7 +162,9 @@ fi
 declare -r version=$1
 declare -r publish_path="${release_server}:${release_path}"
 
-[[ `java -version 2>&1 | grep -E "java version|openjdk version" | cut -d ' ' -f3 | cut -d '.' -f2` -eq 8 ]] || fail "Java version is not 1.8"
+JAVA_VERSION=`java -version 2>&1 | grep -E "java version|openjdk version" | cut -d '"' -f2 | cut -d '.' -f1`
+
+[[ $JAVA_VERSION -ge 9 ]] || fail "Java version is not at least 9"
```

Comment:
I'm not sure I have an up to date overview picture of what this is doing. It requires jdk9 when releasing, and will also use jdk8 rt.jar for some parts? You could explain in today's meeting, but perhaps worth documenting in some comment how it works.